internal/initialize/primitives.go

```diff
@@ -25,12 +25,25 @@ func ByteMap(m *map[string][]byte) {
 	}
 }
 
+// FromPointer returns the value that p points to.
+// When p is nil, it returns the zero value of T.
+func FromPointer[T any](p *T) T {
+	var v T
+	if p != nil {
+		v = *p
+	}
+	return v
+}
+
 // Int32 returns a pointer to v.
 func Int32(v int32) *int32 { return &v }
 
 // Int64 returns a pointer to v.
 func Int64(v int64) *int64 { return &v }
 
+// Pointer returns a pointer to v.
+func Pointer[T any](v T) *T { return &v }
```

Comment:
🤔 Def not suggesting this now, but any reason to keep around non-generic funcs that do this same thing?

Reply:
The functions for numeric types might be convenient with untyped constants; `initialize.Int32(0)` versus `initialize.Pointer(int32(0))`. The latter, "initialize pointer int32," sounds more clear to me at this moment.

https://go.dev/ref/spec#Constants
> The default type of an untyped constant is `bool`, `rune`, `int`, `float64`, `complex128` or `string` respectively, depending on whether it is a boolean, rune, integer, floating-point, complex, or string constant.